puppet/modules/jenkins_job_builder/files/centos.org/ansible/provision.yml

```diff
@@ -10,7 +10,7 @@
       register: cico_get_data
       retries: 5
       delay: 5
-      until: cico_get_data.results.ssid
+      until: cico_get_data.results and cico_get_data.results.ssid
```

Comment:
So I tried this, and it needs to be `cico_get_data.results is defined and cico_get_data.results.ssid` to work. But this also will only work on Ansible < 2.8, as since 2.8 `results` is a reserved keyword and the element is getting renamed to `ansible_module_results` with the following warning:

```
[WARNING]: Found internal 'results' key in module return, renamed to 'ansible_module_results'.
```

Reply:
Opened a PR against cicoclient for the later: https://github.com/CentOS/python-cicoclient/pull/24

Reply:
While hacking the module, I noticed it has a `retry_count` parameter, that defaults to 1, maybe setting this instead of making Ansible do the work for us is nicer?